### ISSUE TITLE
Better topic management

### ISSUE THREAD

**Issue by fdelbrayelle** [fdelbrayelle/generator-jhipster-kafka#30]
##### **Overview of the feature request**

Separate topic name from consumer for instance to update this structure:

```
kafka:
  bootstrap.servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
  polling.timeout: 10000
  consumer:
    fooEntity:
      name: queuing.my_application.foo_entity
      enabled: true
      '[key.deserializer]': org.apache.kafka.common.serialization.StringDeserializer
      '[value.deserializer]': com.mycompany.myapp.service.kafka.deserializer.FooDeserializer
      '[group.id]': my_application
      '[auto.offset.reset]': earliest
    barEntity:
      name: queuing.my_application.bar_entity
      enabled: true
      '[key.deserializer]': org.apache.kafka.common.serialization.StringDeserializer
      '[value.deserializer]': com.mycompany.myapp.service.kafka.deserializer.BarDeserializer
      '[group.id]': my_application
      '[auto.offset.reset]': earliest
```

Into:

```
kafka:
  bootstrap.servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
  polling.timeout: 10000
  topics:
    foo-entity: queuing.my_application.foo_entity
    bar-entity: queuing.my_application.bar_entity
  consumer:
    fooEntity:
      enabled: true
      '[key.deserializer]': org.apache.kafka.common.serialization.StringDeserializer
      '[value.deserializer]': com.mycompany.myapp.service.kafka.deserializer.FooDeserializer
      '[group.id]': my_application
      '[auto.offset.reset]': earliest
    barEntity:
      enabled: true
      '[key.deserializer]': org.apache.kafka.common.serialization.StringDeserializer
      '[value.deserializer]': com.mycompany.myapp.service.kafka.deserializer.BarDeserializer
      '[group.id]': my_application
      '[auto.offset.reset]': earliest
```

In **Big Bang Mode**, previous or existing `application.yml` configuration will be erased so there is no need to ask for which existing topic name.

In **Incremental Mode**, after choosing components a new question "For what topic?" will be added with following answers:
- existing topic names in current configuration
- input a topic name
- default topic name (queuing.application_name.entity_name)

:warning: There is a problem when choosing a topic created for a Foo entity consumer in a new Bar entity consumer?

##### **Motivation for or Use Case**

Currently 1 topic = 1 consumer but in real life more than 1 consumer can consume messages from 1 topic.

##### **Related issues or PR**

- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by fdelbrayelle** [comment]
Linked with https://github.com/fdelbrayelle/generator-jhipster-kafka/issues/17

**Comment by fdelbrayelle** [comment]
Fixed by #86
